packages/ui-tests/cypress/e2e/designer/rootContainerConfig/rootContainersConf.cy.ts

```diff
@@ -90,7 +90,7 @@ describe('Test for camel route root containers configuration', () => {
     cy.uploadFixture('flows/kamelet/basic.yaml');
     cy.openDesignPage();
 
-    cy.openRootConfigurationTab('eip-action');
+    cy.openRootConfigurationTab('Produces periodic events about random users!');
```

Comment:
@tplevko, now we see the Kamelet's `description` in the node's label by default.